src/server/src/projects/entities/update-project.dto.ts

```diff
@@ -18,6 +28,11 @@ export class UpdateProjectDto implements UpdateProjectData {
   @IsBoolean()
   @IsOptional()
   readonly advancedEditingEnabled: boolean;
+  @IsOptional()
+  @IsPositive({ message: "Population deviation must be positive" })
+  @IsNumber()
+  @Max(100, { message: "Population deviation must be between 0% and 100%" })
```

Comment:
As above in `create-project.dto`:
```suggestion
  @Max(100, { message: "Population deviation must be between 0% and 100%" })
  @Min(0, { message: "Population deviation must be between 0% and 100%" })
```